src/components/common/Extensions/SupportingDocuments/consts.ts

```diff
@@ -14,8 +19,14 @@ export const links = {
       text: 'supporting.documents.link1',
     },
     {
-      url: MOTIONS_AND_DISPUTES,
+      url: LAZY_CONSENSUS_EXTENSION,
       text: 'supporting.documents.link3',
     },
   ],
+  [Extension.StreamingPayments]: [
+    {
+      url: STREAMING_PAYMENTS,
+      text: 'supporting.documents.link4',
```

Comment:
This naming is horrible. It's not descriptive at all.

Yes, I know you didn't add it in initially, but it's still bad.